cmake/Platform/Linux/Packaging/o3de.editor.in

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+pushd @CPACK_PACKAGING_INSTALL_PREFIX@
+bin/Linux/profile/Default/Editor $@
+popd
```

Comment:
I would also suggest adding a sanity check for the existence of the binaries in these scripts as well and report an error that it is missing and may need to reinstall (for this and the other scripts)
```suggestion
if [ ! -f @CPACK_PACKAGING_INSTALL_PREFIX@/bin/Linux/profile/Default/Editor ];
then
    echo "Missing O3DE Editor from '@CPACK_PACKAGING_INSTALL_PREFIX@
pushd @CPACK_PACKAGING_INSTALL_PREFIX@
bin/Linux/profile/Default/Editor $@
popd
```